tests/TestRunner/TestRunner.py

```diff
@@ -219,7 +219,7 @@ def processLine(self, line, linenr):
             return None
         handler = XMLParser.XMLHandler()
         errorHandler = XMLParser.XMLErrorHandler()
-        xml.sax.parseString(line, handler, errorHandler=errorHandler)
+        xml.sax.parseString(line.encode('utf-8'), handler, errorHandler=errorHandler)
```

Comment:
so it seems to me that this is a bug in the xml.sax module. In newer (I checked 3.7) python versions, this is how the byte stream is created:

```
    if isinstance(string, str):
        inpsrc.setCharacterStream(io.StringIO(string))
    else:
        inpsrc.setByteStream(io.BytesIO(string))
```

But the good news here is that I don't think this fix will break newer python versions